<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashReporter</class>
 <widget class="QWidget" name="CrashReporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hyves Desktop Error Reporter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hyves Desktop was closed unexpectedly. Please help us solve the problem by sending some information to Hyves.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="uploadCheckbox">
     <property name="toolTip">
      <string>This information includes the type of computer you use, and information about the usage of Hyves Desktop. No personal information is sent to Hyves.</string>
     </property>
     <property name="text">
      <string>&amp;Send information about this problem to Hyves</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="text">
        <string>&amp;Restart Hyves Desktop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uploadCheckbox</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>restartButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
